### PR TITLE
chore(CPlusPlus): add max and min average

### PR DESCRIPTION
**Pull request by manekasingh05**
Maximum, Minimum and average in an array

**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [x] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

**Comment by manekasingh05** [comment]
Mximum, minimum and average

File: algorithms/CPlusPlus/Arrays/max-min-avg.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+int min(int *arr, int k, int size)     //function to find minimum element in an array
+{
+    int i;
+for(i=1; i<size; i++)
+{
+    if(arr[i]<k)
+    {
+        k=arr[i];
+    }
+}
```
Comment (aayushjain7) [inline algorithms/CPlusPlus/Arrays/max-min-avg.cpp:41]: fix indentation.

**Review by aayushjain7 — CHANGES_REQUESTED** [review]
update readme.MD file also

**Review by manekasingh05 — COMMENTED** [review]
Please review changes

**Review by aayushjain7 — CHANGES_REQUESTED** [review]
update readme.MD file also

**Comment by ming-tsai** [comment]
Hi @manekasingh05,
Any updates?

**Comment by manekasingh05** [comment]
> Hi @manekasingh05,
> Any updates?

I have already updated readme.Md file as per the instructions it was mentioned to add in the same file in the form of comments. Do I need to make separate file and keep both of these files in a folder?

**Comment by ming-tsai** [comment]
> > Hi @manekasingh05,
> > Any updates?
> 
> I have already updated readme.Md file as per the instructions it was mentioned to add in the same file in the form of comments. Do I need to make separate file and keep both of these files in a folder?

You should modify on your branch `manekasingh05:patch-3` and the readme.md is on `CPlusPlus` folder

**Comment by ming-tsai** [comment]
Hi @manekasingh05,
Any update?

**Comment by manekasingh05** [comment]
> Hi @manekasingh05,
> Any update?

Yes, updated Readme.md. You can check now!

**Review by aayushjain7 — APPROVED** [review]

File: algorithms/CPlusPlus/README.md [hunk 2]
```diff
@@ -29,6 +29,7 @@
 - [Merge two sorted array without using extraspace](Arrays/merge-two-sorted-array.cpp)
 - [All unique triplet that sum up to given value](Arrays/three-sum.cpp)
 - [Next permutation](Arrays/next-permutation.cpp)
+- [Maximum Minimum Average of numbers](max-min-avg.cpp)
```
Comment (ming-tsai) [inline algorithms/CPlusPlus/README.md:32]: Please fix the link
```suggestion
- [Maximum Minimum Average of numbers](Arrays/max-min-avg.cpp)
```

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**